### PR TITLE
updated `pages/learn/foundations/from-react-to-nextjs/next-steps.mdx`

### PR DESCRIPTION
**Pull request by FoodChain1028**
first update

### CODE REVIEW

File: pages/learn/foundations/from-react-to-nextjs/next-steps.mdx [hunk 1]
```diff
@@ -1,3 +1,11 @@
-import NotTranslateYet from '@components/NotTranslateYet';
+# 下一步
 
-<NotTranslateYet/>
+恭喜你完成了 Next.js 的基礎課程！
+
+若想要繼續學習 Next.js，我們推薦此教學：「[建造你的第一個 Next.js 應用程式](https://nextjs.org/learn/basics/create-nextjs-app?utm_source=next-site&utm_medium=homepage-cta&utm_campaign=next-website)」，在此教學中我們將會介紹 Next.js 的主要功能。
+
+你也可以閱讀[此份文件](https://nextjs.org/docs/getting-started)並瀏覽[這裡](https://nextjs.org/examples)展示的諸多例子。
```
Comment (yuaanlin) [inline pages/learn/foundations/from-react-to-nextjs/next-steps.mdx:7]:
```suggestion
你也可以閱讀[此份文件](/docs/getting-started)並瀏覽[這裡](/examples)展示的諸多例子。
```

**Review by yuaanlin — CHANGES_REQUESTED** [review]
Hi, thanks for the contribution! I think there is a mistake about where this article should be translated at.

To be specific, the original path of the translated article you submitted in this pull request is  https://nextjs.org/learn/foundations/how-nextjs-works/next-steps , so it should be at 

```
/learn/foundations/how-nextjs-works/next-steps.mdx
``` 

but you translated at 

```
/learn/foundations/from-react-to-nextjs/next-steps.mdx
```

Can you help me move the file into the correct path?

**Comment by FoodChain1028** [comment]
I am sorry. I'll change it really soon.

File: pages/learn/foundations/how-nextjs-works/next-steps.mdx [hunk 2]
```diff
@@ -1,3 +1,11 @@
...
-<NotTranslateYet/>
+恭喜你完成了 Next.js 的基礎課程！
+
+若想要繼續學習 Next.js，我們推薦此教學：「[建立你的第一個 Next.js 應用程式](/learn/basics/create-nextjs-app)」，在此教學中我們將會介紹 Next.js 的主要功能。
+
+你也可以閱讀[此份文件](/docs/getting-started)並瀏覽[這裡](/examples)展示的諸多例子。
+
+# 加入社群
+
+若你有任何問題，歡迎加入我們在 [Discord](https://nextjs.org/discord) 的社群或是 [GitHub 的討論版](https://github.com/vercel/next.js/discussions)。
```
Comment (yuaanlin) [inline pages/learn/foundations/how-nextjs-works/next-steps.mdx:11]:
```suggestion
若你有任何問題，歡迎加入我們在 [Discord](https://nextjs.org/discord) 的社群或是 [GitHub 的討論版](https://github.com/vercel/next.js/discussions)。

<TranslatedBy githubUserNames={["FoodChain1028"]} />

```

File: pages/learn/foundations/how-nextjs-works/next-steps.mdx [hunk 3]
```diff
@@ -1,3 +1,11 @@
-import NotTranslateYet from '@components/NotTranslateYet';
+# 下一步
```
Comment (yuaanlin) [inline pages/learn/foundations/how-nextjs-works/next-steps.mdx:1]:
```suggestion
# Next.js 是如何運作的

## 下一步
```

File: pages/learn/foundations/how-nextjs-works/next-steps.mdx [hunk 4]
```diff
@@ -1,3 +1,11 @@
...
 
-<NotTranslateYet/>
+恭喜你完成了 Next.js 的基礎課程！
+
+若想要繼續學習 Next.js，我們推薦此教學：「[建立你的第一個 Next.js 應用程式](/learn/basics/create-nextjs-app)」，在此教學中我們將會介紹 Next.js 的主要功能。
+
+你也可以閱讀[此份文件](/docs/getting-started)並瀏覽[這裡](/examples)展示的諸多例子。
+
+# 加入社群
```
Comment (yuaanlin) [inline pages/learn/foundations/how-nextjs-works/next-steps.mdx:9]:
```suggestion
## 加入社群
```

**Review by yuaanlin — APPROVED** [review]
Thanks!

**Pull request merged**